red_flags.py

```
@@ -59,6 +59,46 @@ def save_to_database(data: Any):
     with open("database.txt", "w") as f:
         f.write(str(data))
 
+# Red Flag 12: Function with too many responsibilities and poor documentation
+def handle_user_request(request_data):
+    # This function does too many things and has no proper documentation
+    # It handles user authentication, data processing, file operations, and email sending
+    # all in one function, making it hard to test and maintain
+    
+    # Authentication
+    if request_data.get('token') != API_KEY:
+        return "Invalid token"
+    
+    # Data processing
```

Reading from a file with open() and no exception type specified in except hides all error types and may mask other problems. Be explicit with exception handling and extract file operations into a helper.